.github/workflows/publish-images.yaml

```diff
@@ -24,6 +24,7 @@ jobs:
         run: |
           grep -v '\#' versions.txt | grep opentelemetry-collector | awk -F= '{print "OTELCOL_VERSION="$2}' >> $GITHUB_ENV
           grep -v '\#' versions.txt | grep targetallocator | awk -F= '{print "TARGETALLOCATOR_VERSION="$2}' >> $GITHUB_ENV
+          grep -v '\#' versions.txt | grep operator-opamp-bridge | awk -F= '{print "OPERATOR_OPAMP_BRIDGE="$2}' >> $GITHUB_ENV
```

Comment:
small nit: can we change the environment variable name here to include `_VERSION` i.e. `OPERATOR_OPAMP_BRIDGE_VERSION` like the other versions being set in this workflow?

Reply:
Done

Reply:
thank you!